software/impler.yml

```diff
@@ -0,0 +1,22 @@
+# software name
+name: "Impler"
+# URL of the software project's homepage
+website_url: "https://impler.io"
+# URL where the full source code of the program can be downloaded
+source_code_url: "https://github.com/implerhq/impler.io"
+# description of what the software does, shorter than 250 characters, sentence case
+description: "Data import experience, built for engineering teams to help them build rich data import experience without constantly reinventing the wheel."
```

Comment:
The description should explain what the software does.

Reply:
Done!